packages/office-ui-fabric-react/src/components/GroupedList/examples/GroupedList.CustomCheckbox.Example.tsx

```diff
@@ -37,54 +34,44 @@ export class GroupedListCustomCheckboxExample extends React.Component<{}, IGroup
           fieldName: key,
           minWidth: 300,
         }),
-      );
-    this._groups = createGroups(groupCount, groupDepth, 0, groupCount);
-
-    this._selection = new Selection();
-    this._selection.setItems(this._items);
-
-    this.state = {};
-  }
+      ),
+  );
+  const selection = useConst(() => {
+    const s = new Selection();
+    s.setItems(items);
+    return s;
+  });
 
-  public render(): JSX.Element {
-    return (
-      <div>
-        <FocusZone>
-          <SelectionZone selection={this._selection} selectionMode={SelectionMode.multiple}>
-            <GroupedList
-              items={this._items}
-              onRenderCell={this._onRenderCell}
-              selection={this._selection}
-              selectionMode={SelectionMode.multiple}
-              groups={this._groups}
-              groupProps={{
-                onRenderHeader: this._onRenderHeader,
-              }}
-            />
-          </SelectionZone>
-        </FocusZone>
-      </div>
-    );
-  }
-
-  private _onRenderHeader = (props: IGroupHeaderProps): JSX.Element => {
-    return <GroupHeader onRenderGroupHeaderCheckbox={this._onRenderGroupHeaderCheckbox} {...props} />;
-  };
-
-  private _onRenderGroupHeaderCheckbox = (props: IGroupHeaderCheckboxProps) => {
-    return <Toggle checked={props.checked} />;
-  };
-
-  private _onRenderCell = (nestingDepth: number, item: IExampleItem, itemIndex: number): JSX.Element => {
-    return (
+  const onRenderCell = React.useCallback(
```

Comment:
You can do `useConstCallback` here since neither `items` nor `selection` will ever change.